entries/g/getresponse.com.json

```diff
@@ -0,0 +1,18 @@
+{
+  "GetResponse": {
+    "domain": "getresponse.com",
+    "tfa": [
+      "sms",
+      "email",
+      "totp",
+      "custom-software"
+    ],
+    "custom-software": [
+      "GetResponse App"
+    ],
+    "documentation": "https://help.elevenlabs.io/hc/articles/21636341747985",
+    "categories": [
+      "marketing"
+    ]
```

Comment:
```suggestion
    ],
    "regions": [
      "-cu",
      "-ir",
      "-kp",
      "-ru",
      "-sy"
    ]
```

From their [Terms of Service](https://www.getresponse.com/legal/terms-us) (typos theirs):
> conform to all applicable laws, regulations, rules and requirements (“Legal Requirements”) relating to the transmission of emails (including but not limited to, commercial emails), creating Content and the processing and transmission of personal data, applicable sanctions, restrictive measures, export and re-export control laws and regulations of the European Union and any of its members, the United States of America, the United Nations (including the governmental authorities of such states), and any other sanctioning authority, relating to economic or trade sanctions, export or trade controls and similar applicable laws, regulations, rules or requirements (“International Sanctions”), including but not limited to the Legal Requirements in force in your country of residence, especially, you represent and warrant that you are not located in country that is subject to any government embargo, or that has been designated as a “terrorist-supporting” country, and you will not directly or indirectly use, sell, supply, export or transfer the Service in any way whatsoever to, on behalf of, or for the benefit of a person or entity located in countries or regions subject to International Sanctions (e.g. Russia) or trade embargoes, or to any person or entity that is otherwise targeted by International Sanctions, or to any person or entity that is directly or indirectly involved in acts of terrorism. You represent that you are not designated, targeted, or subject to International Sanctions in a way that may cause GeResponse to violate International Sanctions, you are not owned or controlled by, and not acting for, or on behalf of, a natural or legal person designated, targeted, or subject to an International Sanctions, in particular, you and such person are not enlisted on any sanctioned entities lists. You shall immediately inform GetResponse in writing if you or any of your beneficiaries is entered on any sanctioned entities lists, as described above;

Reply:
Thank you for the correction. I don’t think I know much about the `regions` item. I will pay more attention to this item in the future! 🌹